src/main/rx/cc2500_frsky_common.h

```diff
@@ -24,13 +24,20 @@
 
 #include "rx/rx_spi.h"
 
+typedef enum {
+  FRSKY_SPI_A1_SOURCE_VBAT = 0,
+  FRSKY_SPI_A1_SOURCE_EXTADC,
+  FRSKY_SPI_A1_SOURCE_CONST
+} frSkySpiA1Source_e;
+
+
 typedef struct rxFrSkySpiConfig_s {
     uint8_t autoBind;
     uint8_t bindTxId[2];
     int8_t  bindOffset;
     uint8_t bindHopData[50];
     uint8_t rxNum;
-    uint8_t useExternalAdc;
+    frSkySpiA1Source_e a1Source;
```

Comment:
Please change this to an `uint8_t`. Enums are stored as `int`, and for parameter groups, size matters.